buildscripts/condarecipe.local/meta.yaml

```
@@ -46,6 +47,7 @@ requirements:
     - numpy >=1.18
     - setuptools
     # On channel https://anaconda.org/numba/
+    - importlib_metadata       # [py<39]
```

```suggestion
    - importlib_metadata       # [py<39]
    # On channel https://anaconda.org/numba/
```
looks like this comment is now on the incorrect line.